pages/bookings/[status].tsx

```diff
@@ -0,0 +1,173 @@
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+      if (!res.ok) {
+        throw new HttpError({ statusCode: res.status });
+      }
+    },
+    {
+      async onSettled() {
+        await utils.invalidateQuery(["viewer.bookings"]);
+      },
+    }
+  );
+  const isUpcoming = new Date(booking.endTime) >= new Date();
+  const isCancelled = booking.status === BookingStatus.CANCELLED;
+
+  const pendingActions = [
+    {
+      id: "confirm",
+      label: "Confirm",
+      onClick: () => mutation.mutate(true),
+      icon: CheckIcon,
+      disabled: mutation.isLoading,
+    },
+    {
+      id: "reject",
+      label: "Reject",
+      onClick: () => mutation.mutate(false),
+      icon: BanIcon,
+      disabled: mutation.isLoading,
+    },
+  ];
+
+  const bookedActions = [
+    {
+      id: "cancel",
+      label: "Cancel",
+      href: `/cancel/${booking.uid}`,
+      icon: XIcon,
+    },
+    {
+      id: "reschedule",
+      label: "Reschedule",
+      href: `/reschedule/${booking.uid}`,
+      icon: ClockIcon,
+    },
+  ];
+
+  return (
+    <tr>
+      <td className={"px-6 py-4" + (booking.rejected ? " line-through" : "")}>
+        {!booking.confirmed && !booking.rejected && (
+          <span className="mb-2 inline-flex items-center px-1.5 py-0.5 rounded-sm text-xs font-medium bg-yellow-100 text-yellow-800">
+            Unconfirmed
+          </span>
+        )}
+        <div className="text-sm text-neutral-900 font-medium  truncate max-w-60 md:max-w-96">
+          {booking.eventType?.team && <strong>{booking.eventType.team.name}: </strong>}
+          {booking.title}
+        </div>
+        <div className="sm:hidden">
+          <div className="text-sm text-gray-900">
+            {dayjs(booking.startTime).format("D MMMM YYYY")}:{" "}
+            <small className="text-sm text-gray-500">
+              {dayjs(booking.startTime).format("HH:mm")} - {dayjs(booking.endTime).format("HH:mm")}
+            </small>
+          </div>
+        </div>
+        {booking.description && (
+          <div className="text-sm text-neutral-600 truncate max-w-60 md:max-w-96">
+            &quot;{booking.description}&quot;
+          </div>
+        )}
+        {booking.attendees.length !== 0 && (
+          <div className="text-sm text-blue-500">
+            <a href={"mailto:" + booking.attendees[0].email}>{booking.attendees[0].email}</a>
+          </div>
+        )}
+      </td>
+      <td className="hidden sm:table-cell px-6 py-4 whitespace-nowrap">
+        <div className="text-sm text-gray-900">{dayjs(booking.startTime).format("D MMMM YYYY")}</div>
+        <div className="text-sm text-gray-500">
+          {dayjs(booking.startTime).format("HH:mm")} - {dayjs(booking.endTime).format("HH:mm")}
+        </div>
+      </td>
+      <td className="px-6 py-4 whitespace-nowrap text-right text-sm font-medium">
+        {isUpcoming && !isCancelled ? (
```

Comment:
this could be `isUpcoming && !isCancelled && {stuff}` instead of `? <stuff> : null` right?

Reply:
Yeah. That can work too